code/drasil-database/lib/Database/Drasil/ChunkDB.hs

```diff
@@ -299,6 +299,13 @@ collectUnits :: Quantity c => ChunkDB -> [c] -> [UnitDefn]
 collectUnits m = map (unitWrapper . flip unitLookup (m ^. unitTable))
  . concatMap getUnits . mapMaybe (getUnitLup m)
 
+-- | Gets abbreviations/acronyms from a list of chunks that have abbreviations.
+collectAbbreviations :: Idea c => [c] -> [c]
+collectAbbreviations = filter hasAbbreviation
+  where hasAbbreviation c = case getA c of
+                              Nothing -> False
+                              Just _  -> True
```

Comment:
I think this should be kept in `-docLang/.../DocumentLanguage.hs` next to your other changes. [`mapMaybe`](https://hackage.haskell.org/package/base-4.21.0.0/docs/Data-Maybe.html#v:mapMaybe) might also benefit you if you intend to use `getA` right after anyways.